.github/workflows/publish-release-synapse-module.yml

```diff
@@ -0,0 +1,76 @@
+# This pipeline runs for every new tag. It will pull the docker container for
+# the commit hash of the tag, and will publish it as `:<tag-name>` and `latest`.
+name: Release Synapse Guest Module
+
+on:
+    push:
+        tags:
+            - "@element-hq/synapse-guest-module@*"
```

Comment:
I think the scoped tag should be `module/restricted-guests/*` or something of that nature to group with the future client component

Reply:
Do you mean to use the same tag/release for both the server & client components?  That's something I'd be in favour of; upstream currently gives a separate tag/release for each piece, which IMO can get unruly & feels less reliable than a single tag/release for both (especially since the server & client must agree on an API).

Reply:
a48dc51

Reply:
Great